store/src/java/com/zimbra/cs/account/soap/SoapProvisioning.java

```diff
@@ -3224,5 +3239,44 @@ public List<HABGroupMember> getHABGroupMembers(Group group) throws ServiceExcept
     public void resetPassword(Account acct, String newPassword, boolean dryRun) throws ServiceException {
         throw ServiceException.UNSUPPORTED();
     }
-
+    
+    /**
+     *
+     * @param status reserved for future use if mail to send for different status to
+     *               be added
+     * @param to     accepts admins email address to which email has to be send
+     * @throws ServiceException if an error occurs while sending email for MDM
+     *                          notification mail
+     */
+    @Override
+    public String sendMdmEmail(String status, String timeInterval) throws ServiceException {
+        String devicesInfo = "";
+        String emailBody = "";
+        GetDeviceStatusResponse resp = invokeJaxb(new GetDeviceStatusRequest(null, null, SyncAdminConstants.MDM_STATUS_SUSPENDED));
+        List<DeviceStatusInfo> statusOfDevices = resp.getDevices().stream()
+                .filter(devices -> DateTimeUtil.checkWithinTime(Timestamp.valueOf(devices.getUpdateTime()),
+                        Integer.parseInt(timeInterval), TimeUnit.MINUTES))
+                .collect(Collectors.toList());
+        int j = 0;
+        for (int i = 0; i < statusOfDevices.size(); i++) {
+                devicesInfo += MailConstants.HTML_TABLE_ROW + MailConstants.HTML_TABLE_DATA + (j + 1)
```

Comment:
https://github.com/Zimbra/zm-mailbox/pull/1319/files#r928165082

Also stringbuilder if there wasn't a better choice than this.

Reply:
done.